### DISCUSSION TITLE
Streaming Video Stops Downloading After Six Seconds

### DISCUSSION THREAD

**Discussion by coloradomountainguy** [aclap-dev/video-downloadhelper#718] [Help and Q&A]
### Your question

My video-downloadhelper just updated to the newest version.  With the new version I can't download more than six seconds from stripchat.com.  It simply stops downloading.  The six seconds I get are perfectly fine, but before the update I could download as long as the stream kept going.  Any insight as to what may have happened?  Thanks for any assistance.  

### Addon & CoApp version and details

version: 9.0.1.22
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"51688f94c1c74cc0"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": true,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_legacy_ui": false,
    "use_sidebar": true,
    "open_count_store": 3,
    "successfull_dl": 5
}

**Comment by GrampaWildWilly** [comment]
We would need the URL of a specific web page.  Telling us Stripchat isn't enough information.  We need the complete URL of a web page on which there is some content you want to download with VDH.

**Comment by coloradomountainguy** [comment]
It's the same pretty much with any of the models but you can try this one as just an example:  

https://stripchat.com/MiaElizabeth

**Reply by GrampaWildWilly** [reply]
At the moment, Firefox is telling me it can't connect to the site, that it might be too busy & I should try again later.  Maybe I'll do that.

It would not surprise me if the content on there is protected by DRM.  It's easy to determine that on Firefox.  I have seen comments on here that neither Chrome not Edge offers the same easy method.  I don't use anything besides Firefox so I can't advise you on the other browsers.  You see, I don't work for VDH.  I'm just another user.  There has been some noise in this forum that it's becoming increasingly common on onlyfans that content is being protected by DRM.  Stripchat could be following the same route.  I don't know.  I'm not a member of either site & I have no plans to become a member.

**Reply by coloradomountainguy** [reply]
I appreciate you looking into it.  There could be some new DRM, but it would be very coincidental that it came out the same time as the VDH update.  Is there a way I could revert back to the prior version of VDH and see if that would solve it?  Thanks!

**Reply by ethanhunt1459** [reply]
Has this been resolved I am still facing the same issue

**Reply by GrampaWildWilly** [reply]
All these months later I still can't connect to the site.  It's stripchat.com.  Spelled exactly like that, right?

By the way, I don't work for VDH.  I'm just another user.

**Reply by ethanhunt1459** [reply]
u can use https://stripchat.global/ or https://superchatlive.com/ they are the same sister sites

**Reply by GrampaWildWilly** [reply]
I took a look at those & I'm seeing the same 6 second issue others are.  The Network Monitor gave no help either.  There are manifests in there but they're all 6 seconds long as well.  I'm stumped.
